td-agent/Rakefile

```diff
@@ -763,6 +764,25 @@ class BuildTask
     end
   end
 
+  def apply_ruby_installer_patches
+    return if BUNDLED_RUBY_INSTALLER_PATCHES.nil?
+
+    version = BUNDLED_RUBY_INSTALLER_X64_VERSION.gsub(/-\d+$/, '')
+    feature_version = BUNDLED_RUBY_INSTALLER_X64_VERSION.gsub(/\d+-\d+$/, '0')
+    ruby_lib_dir = File.join(td_agent_staging_dir, "lib", "ruby", feature_version)
```

Comment:
```
version = BUNDLED_RUBY_INSTALLER_X64_VERSION.split(/-/, 2).first
feature_version = version.sub(/\d+$/, '0')
```

It may be better to apply sub operation to  a reduced target (version)

Reply:
Thanks, I've fixed it.